websub-generic-integration-tests/tests/test_custom_subscribers.bal

```diff
@@ -32,88 +30,80 @@ public type CustomSubMockDomainEvent record {|
     string domain;
 |};
 
-@websub:SubscriberServiceConfig {
-    path:"/key"
-}
-service customSubKeyWebhook on new CustomSubWebhookServerForPayload(23585) {
-    resource function onIntentVerification(websub:Caller caller, websub:IntentVerificationRequest verRequest) {
+@websub:SubscriberServiceConfig {}
```

Comment:
Do we need this `websub:SubscriberServiceConfig` anymore as it's empty?

Reply:
This becomes empty only for these custom subscribers. If we are to use Ballerina SubscriberService which is given by the websub module, this is compulsory to have with certain mandatory config fields. So there is a validation done to check whether this is present.

Reply:
Ack